README.md

```diff
@@ -30,6 +26,17 @@ Check out our NeurIPS 2021 demonstration ["Training Transformers Together"](http
 To learn more about the ideas behind this library,
 see the [full list](https://github.com/learning-at-home/hivemind/tree/refer-to-discord-in-docs#citation) of our papers below.
 
+## Hivemind in Action: Applications and Use Cases
+
+This section lists external projects that leverage hivemind for decentralized training. 
```

Comment:
```suggestion
This section lists projects that leverage hivemind for decentralized training. 
```

Some of these projects can't really be called "external" (in terms of the authors).